src/main/java/task22/ReversMain.java

```diff
@@ -0,0 +1,24 @@
+package task22;
+
+import java.io.*;
+
+public class ReversMain {
+    public static void main(String[] args) {
+        File dir = new File("src/main/java/task.2.2.result");
```

Comment:
This path is used more than once. It makes sense to store it in a constant but not to copy-paste